src/essos/pxWindowNative.cpp

```diff
@@ -536,10 +536,10 @@ void pxWindowNative::runEventLoop()
         }
     }
 
-    rtLogInfo("pxcore framerate: %d", framerate);
+    rtLogDebug("pxcore framerate: %d", framerate);
```

Comment:
Please leave this logging statement as info.  It's important for it to be logged by default at the info level